### PR TITLE
#4: allowing custom values from pages and posts

### PR DESCRIPTION
**Pull request by spekulatius**
Like this @geekplux ?

### CODE REVIEW

**Comment by geekplux** [comment]
Deploy preview for *geekplux* processing.

Building with commit fb581fa774a4a1f2d42a93ca371e8cf83993f50f

https://app.netlify.com/sites/geekplux/deploys/5cea623dc46541000922f3f4

**Comment by egoist** [comment]
hmm set the head in layout instead.

File: packages/saber-plugin-seo/saber-browser.js [hunk 1]
```diff
@@ -2,22 +2,22 @@ export default function({ rootOptions }) {
...
+      const siteConfig = this.$siteConfig;
+      const page = this.$children[0].$page.attributes;
+
       return {
         meta: [
           {
             name: `description`,
-            content: (this.$children[0].$page.attributes.description != '') ?
-              this.$children[0].$page.attributes.description : this.$siteConfig.description
+            content: (page.description != '') ? page.description : siteConfig.description
```
Comment (spekulatius) [inline packages/saber-plugin-seo/saber-browser.js]: I've reworked it a little make the current work-around maybe a little more appealing. What do you think about this ? @geekplux @egoist

**Comment by egoist** [comment]
so this plugin isn't really necessary to me 😅

**Comment by spekulatius** [comment]
> so this plugin isn't really necessary to me sweat_smile

Yeah, that's actually a very good point - should it actually be outside of the theme? It's kinda a feels like something to reconsider in general.

**Comment by spekulatius** [comment]
What do you think we should do @geekplux ? Want to merge it or ditch it?

**Comment by geekplux** [comment]
sorry, I think it's enough using `head()` in layout file to meet your needs. So I agree to @egoist , it's a feature of a theme, not a plugin.

**Comment by spekulatius** [comment]
Alright. That's cool. Makes sense 👍️

**Pull request closed without merging**